packages/cli/src/cli/clean.rs

```diff
@@ -9,8 +11,36 @@ use super::*;
 #[clap(name = "clean")]
 pub struct Clean {}
 
+static N_TO_IEC_UNIT_MAP: Lazy<HashMap<i32, &str>> = Lazy::new(|| {
+    HashMap::<i32, &str>::from([
+        (1, "K"),
+        (2, "M"),
+        (3, "G"),
+        (4, "T"),
+        (5, "P"),
+        (6, "E"),
+        (7, "Z"),
+        (8, "Y"),
+    ])
+});
+
+static IEC_UNIT_TO_N_MAP: Lazy<HashMap<&str, i32>> = Lazy::new(|| {
+    HashMap::<&str, i32>::from([
+        ("B", 0),
+        ("KiB", 1),
+        ("MiB", 2),
+        ("GiB", 3),
+        ("TiB", 4),
+        ("PiB", 5),
+        ("EiB", 6),
+        ("ZiB", 7),
+        ("YiB", 8),
+    ])
+});
+
```

Comment:
Could we replace these runtime-allocated hashmaps with match statements?

E.g.
```rs
match iec {
   "B" => 0,
   "KiB" => 1,
   ...
}
```

Reply:
Huh... I guess we could. Instead of `.get()` function.